newIDE/app/yarn.lock

```diff
@@ -1653,410 +1653,410 @@
-  version "5.3.0"
-  resolved "https://registry.yarnpkg.com/@pixi/filter-blur/-/filter-blur-5.3.0.tgz#adbd49e4c2ce8be0b3c5e9ee1853ff352d068120"
-  integrity sha512-p/Wjy+ylrCkUuMeNneubKkNuw2b9pDxEM9Ox1wAzWwe9MXYNeU/NU3RPnU7d0XGtxK+VxpOm3ktKbghtuMzoJQ==
-  dependencies:
-    "@pixi/core" "5.3.0"
-    "@pixi/settings" "5.3.0"
-
-"@pixi/filter-color-matrix@5.3.0":
-  version "5.3.0"
-  resolved "https://registry.yarnpkg.com/@pixi/filter-color-matrix/-/filter-color-matrix-5.3.0.tgz#b1fe75af978cd25288b5528eeba950f799259d24"
-  integrity sha512-TytHEkVtmVEaEKNYUtbLMxjE66FJsTRLZxdXDfbKJmYSlItvKj/Dfaau8JVntnK5hXCgQShnGZv7pQJmOZGJwQ==
-  dependencies:
-    "@pixi/core" "5.3.0"
-
-"@pixi/filter-displacement@5.3.0":
-  version "5.3.0"
-  resolved "https://registry.yarnpkg.com/@pixi/filter-displacement/-/filter-displacement-5.3.0.tgz#3b09a4ee018f423c55e7c450e03479f20619259b"
-  integrity sha512-fExR6lo0WYLuxB5fWGzzE6v9rHO5Pbq2W54oa1GKCEmKgUsKlRE81TGAeSOfSlz7wilrjQkfOzAECk8Q4b1Isg==
-  dependencies:
-    "@pixi/core" "5.3.0"
-    "@pixi/math" "5.3.0"
-
-"@pixi/filter-fxaa@5.3.0":
-  version "5.3.0"
-  resolved "https://registry.yarnpkg.com/@pixi/filter-fxaa/-/filter-fxaa-5.3.0.tgz#86481331d44715501709234432b15853a22d448c"
-  integrity sha512-vQxDEYGfu7ZF1IE5GpFPcUaOfWTiUq0ZgSdHzTlOdpdQ08Cq8DgTZAxRAd2sh1RBHANpyMlaFgAJLW6zJcLAnQ==
-  dependencies:
-    "@pixi/core" "5.3.0"
-
-"@pixi/filter-noise@5.3.0":
-  version "5.3.0"
-  resolved "https://registry.yarnpkg.com/@pixi/filter-noise/-/filter-noise-5.3.0.tgz#19d103678dc2640ad4d6f3dfee3cdc719ddd5999"
-  integrity sha512-mL3QjZgGBZRmuaT9j8b985dx0d0oO1B34OIV4T8mp8Mgvxt8XtnVNJyttTACRCZE7hoRw7FyiauOwiApNYG3vw==
-  dependencies:
-    "@pixi/core" "5.3.0"
-
-"@pixi/graphics@5.3.0":
-  version "5.3.0"
-  resolved "https://registry.yarnpkg.com/@pixi/graphics/-/graphics-5.3.0.tgz#18123fe4f0f9d2e5349817ed43a28c39a240c80e"
-  integrity sha512-ey+uo6oABO4uGF514LINzte4za2Ezrf0keMpmynaoN8Eafe9Uep/PEmLj2BcdoNugRzcruJLqEoSQusQHQ23Jg==
-  dependencies:
-    "@pixi/constants" "5.3.0"
-    "@pixi/core" "5.3.0"
-    "@pixi/display" "5.3.0"
-    "@pixi/math" "5.3.0"
-    "@pixi/sprite" "5.3.0"
-    "@pixi/utils" "5.3.0"
-
-"@pixi/interaction@5.3.0":
-  version "5.3.0"
-  resolved "https://registry.yarnpkg.com/@pixi/interaction/-/interaction-5.3.0.tgz#f7920ec208771c7be3b956d70db47bf795d06102"
-  integrity sha512-gc3uOzZqfKD9UCjNUYL23F8eLqXF+Epnji/HtwzOFq6P9Ybr5DIrbAG/8nFS7UjHIUUbFZ/Tw5FgqgprfAfDwA==
-  dependencies:
-    "@pixi/core" "5.3.0"
-    "@pixi/display" "5.3.0"
-    "@pixi/math" "5.3.0"
-    "@pixi/ticker" "5.3.0"
-    "@pixi/utils" "5.3.0"
-
-"@pixi/loaders@5.3.0":
-  version "5.3.0"
-  resolved "https://registry.yarnpkg.com/@pixi/loaders/-/loaders-5.3.0.tgz#44424edee635e1d6b597373f30181606f78b7f02"
-  integrity sha512-g85GUuGWjJmh59J8gVjcIIy9rz+Zurj+cgVWb1oGTmpAVn2q19VX9Wks167SW+vLlng4UWJyjOV91pKnW7Ys9g==
-  dependencies:
-    "@pixi/core" "5.3.0"
-    "@pixi/utils" "5.3.0"
+"@pixi/accessibility@5.3.3":
```

Comment:
Is this normal that this yarn.lock is modified? 🤔

Reply:
Nope :sweat_smile: - I've removed those changes now. I should've noticed that before committing, but anyway, learnt about file-specific git commands by reverting this :D